tls-utils/src/tls_certificates.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::arithmetic_side_effects)]
```

Comment:
is this addition needed? isn't the file just moved to a new location?

Reply:
Yes, this is necessary. A similar "allow" was on the crate level in the original crate. Clippy gets confused with benign addition for a buffer size computation otherwise. If it is not benign then please let me know why. I've changed the code to appease clippy without the #allow, though I feel like in this case it was overly protective. 

The bigger problem is the streamer crate from which the code was taken, where the same clippy warning is triggered many many times if the allow line is removed. I can look into that in a different PR.

Reply:
looks like you removed this?